security-analysis/security-analysis-api/src/main/java/com/powsybl/security/SecurityAnalysisFactoryImpl.java

```diff
@@ -14,6 +15,7 @@
 /**
  * @author Geoffroy Jamgotchian <geoffroy.jamgotchian at rte-france.com>
  */
+@AutoService(SecurityAnalysisFactory.class)
```

Comment:
Why do you add this autoservice?

Reply:
Needed to discover the implementation at runtime in Plugins.java no ?
`List<T> pluginImpls = new ServiceLoaderCache<>(pluginInfo.getPluginClass()).getServices();`

Reply:
No, because plugin interface is  PluginInfo and not SecurityAnalysisFactory

Reply:
Let's take AmplExporter an implementation of Exporter as an example, we have ExporterPluginInfo for plugin interface of Exporter; then in each implementation of Exporter, autoservice is used to make the implementation visible at run-time. Do I understand correctly?

Reply:
Done